app/uk/gov/hmrc/preferenceschangednotifier/model/PreferencesChanged.scala

```diff
@@ -43,21 +43,20 @@ object PreferencesChanged {
   def from(pcRequest: PreferencesChangedRequest): PreferencesChanged =
     PreferencesChanged(
       _id = new ObjectId(),
-      entityId = pcRequest.entityId,
+      entityId = EntityId(pcRequest.entityId),
       changedValue = pcRequest.changedValue,
       preferenceId = new ObjectId(pcRequest.preferenceId),
       updatedAt = pcRequest.updatedAt,
       taxIds = pcRequest.taxIds,
       bounced = pcRequest.bounced
     )
 
-  implicit val objectIdFormat: Format[ObjectId] = MongoFormats.objectIdFormat
-  implicit val dtf: Format[Instant] = MongoJavatimeFormats.instantFormat
-  implicit val mdf: Format[MessageDeliveryFormat] = MessageDeliveryFormat.format
+  given Format[ObjectId] = MongoFormats.objectIdFormat
+  given Format[Instant] = MongoJavatimeFormats.instantFormat
+  given Format[EntityId] = EntityId.given_Format_EntityId
+  given Format[MessageDeliveryFormat] = MessageDeliveryFormat.given_Format_MessageDeliveryFormat
 
-  implicit val reads: Reads[PreferencesChanged] = Json.reads[PreferencesChanged]
-  implicit val writes: OWrites[PreferencesChanged] =
-    Json.writes[PreferencesChanged]
-  implicit val format: OFormat[PreferencesChanged] =
-    Json.format[PreferencesChanged]
+  given Reads[PreferencesChanged] = Json.reads[PreferencesChanged]
```

Comment:
suggestion - I think You don’t need to define all three typeclass instances (Reads, OWrites, and OFormat) explicitly here. The OFormat instance already includes both the Reads and OWrites